rotkehlchen/db/minimized_schema.py

```diff
@@ -17,7 +17,7 @@
     'manually_tracked_balances': 'idINTEGERPRIMARYKEY,assetTEXTNOTNULL,labelTEXTNOTNULL,amountTEXT,locationCHAR(1)NOTNULLDEFAULT("A")REFERENCESlocation(location),categoryCHAR(1)NOTNULLDEFAULT("A")REFERENCESbalance_category(category),FOREIGNKEY(asset)REFERENCESassets(identifier)ONUPDATECASCADE',
     'trades': 'idTEXTPRIMARYKEYNOTNULL,timestampINTEGERNOTNULL,locationCHAR(1)NOTNULLDEFAULT("A")REFERENCESlocation(location),base_assetTEXTNOTNULL,quote_assetTEXTNOTNULL,typeCHAR(1)NOTNULLDEFAULT("A")REFERENCEStrade_type(type),amountTEXTNOTNULL,rateTEXTNOTNULL,feeTEXT,fee_currencyTEXT,linkTEXT,notesTEXT,FOREIGNKEY(base_asset)REFERENCESassets(identifier)ONUPDATECASCADE,FOREIGNKEY(quote_asset)REFERENCESassets(identifier)ONUPDATECASCADE,FOREIGNKEY(fee_currency)REFERENCESassets(identifier)ONUPDATECASCADE',
     'evm_transactions': 'tx_hashBLOBNOTNULL,chain_idINTEGERNOTNULL,timestampINTEGERNOTNULL,block_numberINTEGERNOTNULL,from_addressTEXTNOTNULL,to_addressTEXT,valueTEXTNOTNULL,gasTEXTNOTNULL,gas_priceTEXTNOTNULL,gas_usedTEXTNOTNULL,input_dataBLOBNOTNULL,nonceINTEGERNOTNULL,PRIMARYKEY(tx_hash,chain_id)',
-    'evm_internal_transactions': 'parent_tx_hashBLOBNOTNULL,chain_idINTEGERNOTNULL,trace_idINTEGERNOTNULL,timestampINTEGERNOTNULL,block_numberINTEGERNOTNULL,from_addressTEXTNOTNULL,to_addressTEXT,valueTEXTNOTNULL,FOREIGNKEY(parent_tx_hash,chain_id)REFERENCESevm_transactions(tx_hash,chain_id)ONDELETECASCADEONUPDATECASCADE,PRIMARYKEY(parent_tx_hash,chain_id,trace_id)',
+    'evm_internal_transactions': 'parent_tx_hashBLOBNOTNULL,chain_idINTEGERNOTNULL,trace_idINTEGERNOTNULL,timestampINTEGERNOTNULL,block_numberINTEGERNOTNULL,from_addressTEXTNOTNULL,to_addressTEXT,valueTEXTNOTNULL,FOREIGNKEY(parent_tx_hash,chain_id)REFERENCESevm_transactions(tx_hash,chain_id)ONDELETECASCADEONUPDATECASCADE,PRIMARYKEY(parent_tx_hash,chain_id,trace_id,from_address,to_address,value)',
```

Comment:
Shouldn't this also go in the packaged DB? we need to remember it. I'll add a note

Reply:
Why does this need to go in the packaged DB? I don't understand what you are talknig about

Reply:
The failed test probably shows me you meant to write about hETH? That's a different file ...